packages/enhanced/src/lib/startup/StartupHelpers.ts

```diff
@@ -20,12 +19,21 @@ const { isSubset } = require(
 const { getAllChunks } = require(
   normalizeWebpackPath('webpack/lib/javascript/ChunkHelpers'),
 ) as typeof import('webpack/lib/javascript/ChunkHelpers');
+const HotUpdateChunk = require(
+  normalizeWebpackPath('webpack/lib/HotUpdateChunk'),
+) as typeof import('webpack/lib/HotUpdateChunk');
+const { getUndoPath } = require(
+  normalizeWebpackPath('webpack/lib/util/identifier'),
+) as typeof import('webpack/lib/util/identifier');
 
 const EXPORT_PREFIX = `var ${RuntimeGlobals.exports} = `;
 
 export const federationStartup = 'federation-entry-startup';
```

Comment:
Since you set the constant , can you change the [line](https://github.com/module-federation/core/pull/2919/files#diff-c553537cb35cd360f9c835df10ba71900a13407fee6feaba4352a8b88858b743R65) value as the constant ?

Reply:
done, good find!